src/hardware/setup.rs

```diff
@@ -118,7 +124,7 @@ pub fn setup(
         .use_hse(8.mhz())
         .sysclk(400.mhz())
         .hclk(200.mhz())
-        .per_ck(100.mhz())
+        .per_ck(50.mhz())
```

Comment:
Why not 100 MHz?

Reply:
no reason really, I was just playing around with different combinations